### PR TITLE
Option for runtime error verification was added 

### PR DESCRIPTION
**Pull request by Maikol1804**
Checkbox to allow runtime verification was added to INGInious' frontend

### CODE REVIEW

File: inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html [hunk 1]
```diff
@@ -58,6 +58,19 @@
     </div>
 </div>
 
+<div class="form-group">
+    <label for="treat_non_zero_as_runtime_error" class="col-sm-4 control-label">Verify runtime error</label>
```
Comment (amrondonp) [inline inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html]: I think the message to the user should me something more descriptive, perhaps "Enforce program to return with status 0" or maybe you have better names? @0xDCA @larranaga ?

File: inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html [hunk 2]
```diff
@@ -58,6 +58,19 @@
...
+
+    <div class="col-sm-8">
+        <div class="checkbox"><label>
+            <input type="checkbox" id="treat_non_zero_as_runtime_error" name="treat_non_zero_as_runtime_error"
+                   $if task_data.get('treat_non_zero_as_runtime_error', False):
```
Comment (amrondonp) [inline inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html]: Maybe we should leave the default actions as to validate return codes (True)?

File: inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html [hunk 3]
```diff
@@ -58,6 +58,19 @@
...
+                       checked="checked"
+            />&nbsp;
```
Comment (0xDCA) [inline inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html]: You use `<input>` instead of `<input />` in HTML5

**Review by 0xDCA — COMMENTED** [review]
Good. A minor comment and we're good to go

File: inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html (hunk 1 above)
Comment (0xDCA) [reply inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html]: "Treat non-zero return codes as runtime errors"

Comment (larranaga) [reply inginious/frontend/webapp/templates/course_admin/edit_tabs/grader.html]: How about?  

`Do not verify output with runtime error`

**Comment by amrondonp** [comment]
LGTM

**Pull request merged**